esp/src/eclwatch/DFUQueryWidget.js

```diff
@@ -649,6 +651,7 @@ define([
 
         initWorkunitsGrid: function () {
             var context = this;
+            var formatNum = hpccCommon.format(",.2f");
```

Comment:
formatNum declaration should be moved to module scope or the Util module as "skewFormat"?